lib/routes/isabelle-dev/blog.ts

```diff
@@ -0,0 +1,60 @@
+import { Route } from '@/types';
```

Comment:
I understand that in principle, the namespace should be the second-level domain. However, in this case, isabelle-dev is a website on its own. The website currently mounts at a subdomain of a personal domain, and this situation can easily change. Hence it might be more suitable to use just `isabelle-dev`.